app/src/main/java/com/infomaniak/swisstransfer/ui/screen/main/transferdetails/TransferDetailsScreen.kt

```diff
@@ -112,51 +111,81 @@ fun TransferDetailsScreen(
                 },
             )
         }
+
+        QrCodeBottomSheet(
+            isVisible = { isQrCodeVisible },
+            transferUrl = transferUrl,
+            closeBottomSheet = { isQrCodeVisible = false },
+        )
+        PasswordBottomSheet(
+            isVisible = { isPasswordVisible },
+            transferPassword = transferPassword,
+            closeBottomSheet = { isPasswordVisible = false },
+        )
     }
 }
 
 @Composable
-private fun TransferMessage(transfer: TransferUi, transferSenderEmail: String?) {
+private fun ColumnScope.FilesList(
+    transfer: TransferUi,
+    recipients: List<String>,
+    isMultiselectOn: Boolean,
+    transferDetailsViewModel: TransferDetailsViewModel,
+) {
 
-    if (transferSenderEmail == null && transfer.message == null) return
+    val shouldDisplayRecipients = recipients.isNotEmpty()
+    val shouldDisplayMessage = transfer.message != null
 
-    Spacer(modifier = Modifier.height(Margin.Large))
+    FileItemList(
+        modifier = Modifier
+            .weight(1.0f)
+            .padding(horizontal = Margin.Medium),
+        files = transfer.files,
+        isRemoveButtonVisible = false,
+        isCheckboxVisible = { isMultiselectOn },
+        isUidChecked = { fileUid -> transferDetailsViewModel.checkedFiles[fileUid] ?: false },
+        setUidCheckStatus = { fileUid, isChecked -> transferDetailsViewModel.checkedFiles[fileUid] = isChecked },
+        header = {
+            Column {
+                Spacer(modifier = Modifier.height(Margin.Large))
+                TransferInfo(transfer)
+                Spacer(modifier = Modifier.height(Margin.Large))
+                if (shouldDisplayRecipients) TransferRecipients(recipients)
+                if (shouldDisplayRecipients && shouldDisplayMessage) Spacer(modifier = Modifier.height(Margin.Mini))
+                if (shouldDisplayMessage) TransferMessage(transfer)
+                if (shouldDisplayRecipients || shouldDisplayMessage) Spacer(modifier = Modifier.height(Margin.Large))
```

Comment:
We can centralize `Modifier.height(Margin.Large)` in a `val` here.

Reply:
Easier to read if all the Spacers share the same writing format.

And easier to update when the UI changes if we can directly access the Margin.

Reply:
They'll have the same writing format :

 ```Spacer(modifier = spacerModifier)```

All spacers would normally have the same size.

And from a `Compose` point of view, I just asked @sirambd and he said it's better to create a variable instead of duplicating the same modifier.